### PR TITLE
1895 scan entity

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

**Review by matthopson — APPROVED** [review]

File: shared/src/business/entities/Scan.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+/**
+ * constructor
+ * @param rawScan
+ * @constructor
+ */
+function Scan({ applicationContext, rawScan }) {
+  Object.assign(this, rawScan, {
+    batches: rawScan.batches || [],
+    createdAt: rawScan.createdAt || new Date().toISOString(),
```
Comment (ericsorenson) [inline shared/src/business/entities/Scan.js]: We should use the date utility to get the ISO string so it's in EST

File: shared/src/business/entities/Scan.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+const _ = require('lodash');
```
Comment (ericsorenson) [inline shared/src/business/entities/Scan.js]: can we import only the remove function here?

File: shared/src/business/entities/Batch.js [hunk 3]
```diff
@@ -2,16 +2,18 @@ const joi = require('joi-browser');
...
  * constructor
  * @param rawBatch
  * @constructor
  */
 function Batch({ applicationContext, rawBatch }) {
+  this.applicationContext = applicationContext;
   this.batchId = rawBatch.batchId || applicationContext.getUniqueId();
   this.batchIndex = rawBatch.batchIndex || 0;
-  this.createdAt = rawBatch.createdAt || new Date().toISOString();
+  this.createdAt = rawBatch.createdAt || createISODateString();
```
Comment (ericsorenson) [inline shared/src/business/entities/Batch.js:16]: 🔥

**Review by ericsorenson — APPROVED** [review]

**Review by matthopson — APPROVED** [review]

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**